### PR TITLE
Adding Lab forms,Enable ANC form to be deployed by Aijar

### PR DESCRIPTION
**Pull request by slubwama**
Adding Lab forms
Changing answers to answerConceptIds

### CODE REVIEW

File: omod/src/main/webapp/resources/htmlforms/Lab_Immunology.html [hunk 1]
```diff
@@ -0,0 +1,153 @@
...
+                    </td>
+                    <td>%</td>
+                </tr>
+
+                <tr>
+                    <td>CD3 Count:</td>
+                    <td>
+                        <obs conceptId="1028"/>
+                    </td>
+                    <td>cells/Ã‚ÂµL</td>
```
Comment (ssmusoke) [inline omod/src/main/webapp/resources/htmlforms/Lab_Immunology.html]: @slubwama Fix this consistently across all the forms

**Comment by ssmusoke** [comment]
@slubwama 

1. See cells/Ã‚ÂµL (special text across multiple forms) 
2. Change the file naming to match the Java camel case naming conventions used for other forms like LabImmunology.html
3. Why are the lab forms html extension files instead of xml? Make these consistent 
4. Since these are simple forms, can you look into making them follow the OpenMRS style guide see https://talk.openmrs.org/t/solved-styling-html-form-entry-forms-to-match-new-ref-app-ui-style-guide/7403 which will form the basis for the short forms at least 
5. Disable the ability to enter multiple encounters on the same date see Trello card https://trello.com/c/HJPyojtk

**Comment by ssmusoke** [comment]
@slubwama What is the purpose of the metadata tag class="simple-form-ui"

2. How about Disable the ability to enter multiple encounters on the same date see Trello card https://trello.com/c/HJPyojtk

**Comment by slubwama** [comment]
@ssmusoke I just forgot to include it in the comments but its one of the changes I made. Please check

**Comment by slubwama** [comment]
Its to implement simple UI other than standard UI

**Comment by ssmusoke** [comment]
@slubwama 

1. Hematology form does not display 
2. Immunology and hematology forms have funny non-Latin characters non-latin characters in forms

**Comment by slubwama** [comment]
@ssmusoke the non-Latin characters are as per the pervious form.

**Pull request merged**